src/trio_monitor/monitor.py

```diff
@@ -58,7 +54,7 @@ def __init__(self):
         self._is_monitoring = False
         # semi-arbitrary size, because otherwise we'll be dropping events
         # no clue how to make this better, alas.
-        self._monitoring_queue = Queue(capacity=100)
+        self._rx, self._tx = open_memory_channel(math.inf)
```

Comment:
don't really like using ``math.inf`` here, i specifically did only use a small-ish queue size to prevent it growing entirely unbounded.